magma/__init__.py

```diff
@@ -79,7 +79,7 @@ def set_mantle_target(t):
 from magma.family import get_family
 _Family_ = get_family()
 
-from .generator import Generator, Generator2
+from .generator import Generator, Generator2, DebugGenerator
```

Comment:
For uniformity sake, I'd say let's call it DebugGenerator2. That way we can just Generator2 -> Generator everywhere once we switch